### PR TITLE
Website: Use Inconsolata as the font for all code samples.

### PR DESCRIPTION
**Pull request by nicksay**

### CODE REVIEW

**Comment by awbraunstein** [comment]
lgtm

File: web/assets/styles/main.css [hunk 1]
```diff
@@ -619,3 +631,9 @@ main nav {
   font-weight: 700;
   src: local('Roboto Condensed Bold'), local('RobotoCondensed-Bold');
 }
+@font-face {
+  font-family: 'Inconsolata';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Inconsolata Regular'), local('Inconsolata-Regular');
```
Comment (rviscomi) [inline web/assets/styles/main.css]: What's the advantage of defining local sources here in addition to the Google Fonts CSS?

Comment (nicksay) [inline web/assets/styles/main.css]: If you have the fonts locally installed and are offline, things still look correct.  (This is particularly good for development.)

Comment (rviscomi) [inline web/assets/styles/main.css]: Cool SGTM. Hadn't thought of that.

**Pull request merged**